### PR TITLE
Released fix for leap year bug

### PR DESCRIPTION
**Pull request by colinbruce**
* refactored into the applicant model
* test for leap year correct response
* test for delegation to applicant model

[ticket](https://www.pivotaltracker.com/story/show/106838812)
[finishes #106838812]

### CODE REVIEW

File: spec/models/application_spec.rb [hunk 1]
```diff
@@ -245,6 +247,20 @@
     end
   end
 
+  describe '#age' do
```
Comment (novotnyjakub) [inline spec/models/application_spec.rb]: We should test this method in `Applicant` model.

File: spec/models/application_spec.rb [hunk 2]
```diff
@@ -245,6 +247,20 @@
...
+    subject { application.applicant_age }
+
+    context 'when applicant is born on Feb 29th in a leap year' do
```
Comment (novotnyjakub) [inline spec/models/application_spec.rb]: Is one single scenario enough for all cases? That's just a question, I'm not sure.

File: app/models/application.rb [hunk 3]
```diff
@@ -151,18 +152,13 @@ def full_name
   end
 
   def applicant_over_61?
-    applicant_age >= 61
+    applicant.age >= 61
```
Comment (novotnyjakub) [inline app/models/application.rb]: I'm thinking whether we do use the `age` somewhere outside this `applicant_over_61?` method. If not, than we don't actually need the delegation. I thought originally that we use the age directly.

Comment (colinbruce) [inline app/models/application.rb]: We are using it a couple of places; a partial view and a service object.

File: spec/models/application_spec.rb (hunk 1 above)
Comment (colinbruce) [inline spec/models/application_spec.rb]: Noted.  I created the test before the refactoring work.

File: app/models/application.rb (hunk 3 above)
Comment (novotnyjakub) [inline app/models/application.rb]: :+1:

**Pull request merged**